selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/webdriver/SeleniumWebDriverHelper.java

```diff
@@ -525,6 +525,32 @@ public String waitVisibilityAndGetAttribute(WebElement element, String attribute
     return waitVisibility(element, timeout).getAttribute(attribute);
   }
 
+  /**
+   * Waits visibility of element which specified by {@code elementLocator} and gets provided {@code
+   * cssPropertyName} from it.
+   *
+   * @param elementLocator locator of the investigated element
+   * @param cssPropertyName name of the css properties which should be extracted
+   * @return value of the specified {@code cssPropertyName}
+   */
+  public String waitAndGetCss(By elementLocator, String cssPropertyName) {
```

Comment:
`waitAndGetCss` sounds too general. `waitAndGetCssProperty` would be clear in this case, IMHO.